js/index.js

```diff
@@ -0,0 +1,91 @@
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const bookContainer = document.querySelector('.book-list');
+const form = document.getElementById('form');
+
+// booksArray to store array of object
+let books = [];
+
+// Create a js Object to store form data
+const formData = {
+  title: '',
+  author: '',
+};
+
+function removeElementFromUI(target) {
+  if (target.classList.contains('remove-btn')) {
+    target.parentElement.remove();
+  }
+}
+
+function removeFromBooks(title) {
+  let tempBooks;
+  if (localStorage.getItem('books') === null) {
+    tempBooks = [];
+  } else {
+    tempBooks = JSON.parse(localStorage.getItem('books'));
+    tempBooks.forEach((book, index) => {
+      if (book.title === title) {
+        tempBooks.splice(index, 1);
+        localStorage.setItem('books', JSON.stringify(tempBooks));
+      }
+    });
+  }
+}
+
+// display data in the browser
+function displayBook(index) {
+  const list = document.createElement('div');
+  list.classList.add('single-book');
+
+  const title = document.createElement('h3');
+  title.textContent = books[index].title;
+  title.classList.add('title');
+  list.appendChild(title);
+
+  const author = document.createElement('h3');
+  author.textContent = books[index].author;
+  author.classList.add('author');
+  list.appendChild(author);
+
+  const removeBtn = document.createElement('button');
+  removeBtn.classList.add('remove-btn');
+  removeBtn.textContent = 'Remove';
+  removeBtn.setAttribute('type', 'button');
+  list.appendChild(removeBtn);
+
+  const bar = document.createElement('hr');
+  bar.classList.add('bar');
+  list.appendChild(bar);
+
```

Comment:
- [ ] Data preserved in the browser's memory localStorage is not being displayed on reload. you should iterate through the list of books got from the localStorage and display them on reload. Kindly make sure that on reload, the saved books are displayed.